### ISSUE TITLE
Investigate legal framework

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.org#11]
We've received an email asking us to close a Liberapay account which is allegedly illegal.

Here is the gist of the message:

> Under French law, it is 100%  illegal to facilitate financial transactions for Holocaust deniers and hate speech advocates such as the following individual, who is also currently a criminal fugitive in Eastern Europe to avoid United States federal proesecutors:

I have replied that we will investigate the validity of those claims, and notified the accused user.

---

Nous avons reçu un courriel nous demandant de fermer un compte Liberapay supposément illégal.

Le message accuse le détenteur du compte de négationnisme et d'incitation à la haine, affirme que la loi française nous interdit de faciliter le transfert d'argent vers de tels individus, et ajoute que l'accusé est un fugitif aux yeux de la justice des États-Unis et qu'il se cache en Europe de l'Est.

J'ai répondu que nous allions vérifier tout ça, et notifié l'accusé.

**Comment by Changaco** [comment]
The accused has responded that he does not deny the Holocaust and is not a fugitive.

---

L'accusé a répondu qu'il ne nie pas l'Holocauste et qu'il n'est pas un fugitif.

**Comment by Changaco** [comment]
New message from the accuser:

> Google "[name of the accused]" and you will find countless news stories published in reputable mainstream US and international media outlets re: his illegally spreading hate propaganda via hacking fax machines, as well as many other additional published reports on his being a fugitive in Eastern Europe. Read his Wikipedia article, Youtube, Twitter and blogs for additional proof.
> 
> French laws on Holocaust Denial:
> 
> In France, the Gayssot Act, voted for on July 13, 1990, makes it illegal to question the existence of crimes that fall in the category of crimes against humanity as defined in the London Charter of 1945, on the basis of which Nazi leaders were convicted by the International Military Tribunal at Nuremberg in 1945–46. When the act was challenged by Robert Faurisson, the Human Rights Committee upheld it as a necessary means to counter possible antisemitism.
> 
>>    LAW No 90-615 to repress acts of racism, anti-semitism and xenophobia (1990)
>>
>>    MODIFICATIONS OF THE LAW OF JULY 29, 1881 ON THE FREEDOM OF THE PRESS Art 8. – Article 24 of the Law on the Freedom of the Press of 29 July 1881 is supplemented by the following provisions: In the event of judgment for one of the facts envisaged by the preceding subparagraph, the court will be able moreover to order: Except when the responsibility for the author of the infringement is retained on the base for article 42 and the first subparagraph for article 43 for this law or the first three subparagraphs for article 93-3 for the law No 82-652 for July 29, 1982 on the audio-visual communication, the deprivation of the rights enumerated to the 2o and 3o of article 42 of the penal code for imprisonment of five years maximum;
>>
>>    Art 9. – As an amendment to Article 24 of the law of July 29, 1881 on the freedom of the press, article 24 (a) is as follows written: <<Art. 24 (a). - those who have disputed the existence of one or more crimes against humanity such as they are defined by Article 6 of the statute of the international tribunal military annexed in the agreement of London of August 8, 1945 and which were a carried out either by the members of an organization declared criminal pursuant to Article 9 of the aforementioned statute, or by a person found guilty such crimes by a French or international jurisdiction shall be punished by one month to one year's imprisonment or a fine.
>>
>>    Art 13. – It is inserted, after article 48-1 of the law of July 29, 1881 on the freedom of the press, article 48-2 thus written: <<Art. 48-2. - publication or publicly expressed opinion encouraging those to whom it is addressed to pass a favourable moral judgment on one or more crimes against humanity and tending to justify these crimes (including collaboration) or vindicate their perpetrators shall be punished by one to five years' imprisonment or a fine.[28]
> 
> 
> TL;DR: You are financially enabling a US fugitive and antisemite Holocaust denier guilty of hate crimes which are prohibited by French law. Thank you for attending to this matter.

**Comment by Changaco** [comment]
If like me you wondered where the author got that translation of French law, it's from [Laws against Holocaust denial - Wikipedia](https://en.wikipedia.org/wiki/Laws_against_Holocaust_denial#France).

**Comment by Changaco** [comment]
The freedom of speech laws are not relevant in this case since the accused has not posted any potentially-illegal content on the Liberapay website. Even if that was the case, it would not be enough to justify closing the account, we would only have to remove the problematic content, and prevent the user from re-posting it.

**Comment by Changaco** [comment]
I've done a bit of research into financial regulations, they're focused on money laundering and the financing of terrorism, with additional provisions concerning tax fraud. None of that applies here, and even if it did the only obligation I've found is to report the suspicious transactions to the proper authorities.

---

Je me suis penché un peu sur les régulations financières, elles sont centrées sur le blanchiment d'argent et le financement du terrorisme, avec quelques trucs en plus concernant la fraude fiscale. Rien de tout ça n'est applicable dans le cas présent, et même si c'était le cas la seule obligation que j'ai trouvé est de déclarer les transactions suspectes aux autorités compétentes.

Des documents de synthèse sont disponibles à partir du [site de l'ACPR](https://acpr.banque-france.fr/), notamment [celui-ci sur les obligations de déclaration](https://acpr.banque-france.fr/fileadmin/user_upload/acp/publications/registre-officiel/201512-Lignes-directrices-ACPR-Tracfin-obligations-declaration.pdf), et [ces recueils de textes](https://cclrf.banque-france.fr/accueil/publications/recueils-de-textes.html). Les textes de lois sont évidemment disponibles sur legifrance, on y trouve notamment [ce chapitre du code monétaire et financier](https://www.legifrance.gouv.fr/affichCode.do?idSectionTA=LEGISCTA000020196709&cidTexte=LEGITEXT000006072026) intitulé « Obligations relatives à la lutte contre le blanchiment des capitaux et le financement du terrorisme », ainsi que [le suivant](https://www.legifrance.gouv.fr/affichCode.do?idSectionTA=LEGISCTA000020196819&cidTexte=LEGITEXT000006072026) intitulé « Obligations relatives au gel des avoirs ».

**Comment by Changaco** [comment]
We now have a user who is receiving a significant amount of money and who is unequivocally antisemite. Completing the study of the legal framework is now a priority.

---

Nous avons désormais un utilisateur qui reçoit un montant non négligeable de dons et qui est clairement antisémite. Achever l'étude du cadre légal est désormais une priorité.

**Comment by Changaco** [comment]
What I have in mind for the rest of the investigation is the following question: could Liberapay be prosecuted as an accomplice (or facilitator) of crimes or offenses committed by a user of the platform?

I think we should also look into laws specific to antisemitism and nazism, even though they're unlikely to concern us.

---

Ce que j'ai à l'esprit pour la suite de l'étude est la question suivante : Liberapay pourrait-elle être poursuivie en tant que complice (ou facilitateur) de crimes ou délits commis par un utilisateur de la plateforme ?

Je pense qu'on devrait aussi se pencher sur les lois spécifiques à l'antisémitisme et au nazisme, même s'il est peu probable qu'elles nous concernent.

**Comment by Changaco** [comment]
We've received a letter accusing the owner of https://liberapay.com/archiveis of multiple crimes and threatening to sue Liberapay if we don't ban him.

**Comment by jorgesumle** [comment]
> We've received a letter accusing the owner of https://liberapay.com/archiveis of multiple crimes and threatening to sue Liberapay if we don't ban him.

Do they have any proof?

**Comment by Changaco** [comment]
> Do they have any proof?

They sent some evidence, but it isn't obvious that Liberapay can ban archive.is without violating article 10 of its bylaws (#40). What's clear is that there's plenty of problematic content on archive.is. I burned some of the printed screenshots I received with the letter because I didn't want to keep them.
